src/test/kotlin/lotto/domain/LottoRankTest.kt

```diff
@@ -0,0 +1,35 @@
+package lotto.domain
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+internal class LottoRankTest {
+
+    private lateinit var winningLottoNumber: List<Int>
+
+    @BeforeEach
+    fun setup() {
+        winningLottoNumber = listOf(1, 2, 3, 4, 5, 6)
+    }
```

Comment:
이런 경우 BeforeEach 없이 아래와 같이 선언해도 사용할 수 있습니다.

```suggestion
    private val winningLottoNumber = listOf(1, 2, 3, 4, 5, 6)
```